### PR TITLE
Fix call gas limit

### PR DESCRIPTION
**Pull request by Vid201**
This PR fixes estimation for call gas limit.

Code is a little bit messy, gonna refactor with some other things in the subsequent PR.

### CODE REVIEW

File: crates/uopool/src/canonical/sanity_check.rs [hunk 1]
```diff
@@ -459,6 +493,7 @@ mod tests {
 
     use super::*;
 
+    #[ignore]
```
Comment (zsluedem) [inline crates/uopool/src/canonical/sanity_check.rs:496]: https://github.com/Vid201/aa-bundler/actions/runs/5166205480/jobs/9306373395
This is stopping me merge this pr.

**Review by zsluedem — APPROVED** [review]
Looks good to me

File: crates/uopool/src/canonical/sanity_check.rs (hunk 1 above)
Comment (zsluedem) [reply crates/uopool/src/canonical/sanity_check.rs:496]: It would be great to mock the case locally instead of relying on external source in test cases.

Comment (Vid201) [reply crates/uopool/src/canonical/sanity_check.rs:496]: Yeah, agree. I was thinking of removing this test case for now and revisiting it later. You can remove it in your PR.

**Pull request merged**